hr_timesheet_predefined_description/views/hr_timesheet_views.xml

```diff
@@ -0,0 +1,37 @@
+<odoo>
+    <record id="view_analytic_line_form_inherit" model="ir.ui.view">
+        <field name="name">account.analytic.line.form.inherit</field>
+        <field name="model">account.analytic.line</field>
+        <field name="inherit_id" ref="hr_timesheet.hr_timesheet_line_tree" />
+        <field name="arch" type="xml">
+            <field name="name" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </field>
+            <field name="name" position="after">
+                <field name="predefined_description_id" />
```

Comment:
As said before, this shouldn't be enforced here in this module, but on custom part. The OCA part must include the m2o, but let the free text one as it was.

Reply:
Hello, I have removed the invisible from the name field and added optional="hide" so that both fields can be used freely. Is this correct? @pedrobaeza